src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/PublicationSearchResultViewModel.cs

```diff
@@ -9,6 +9,7 @@ namespace GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels
 public record PublicationSearchResultViewModel
```

Comment:
Just wondering if we need to add the legacy publication url to this view model? Not sure if it's needed with the new design but thought it'd be worth mentioning

Reply:
Thanks, legacy publications "Not yet on this service" aren't going to be displayed, or at least not in the current plan. 

@cjrace is planning to have them all removed from the service by the time the new Find Statistics page goes live.

I think there's currently 5, with 2 due to be removed imminently.

`ListPublications` has been designed to only return publications with published releases.